app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -319,7 +319,6 @@ class BrowserTabViewModel(
     private lateinit var tabId: String
     private var webNavigationState: WebNavigationState? = null
     private var httpsUpgraded = false
-    private var shouldShowNewProgress = true
```

Comment:
@marcosholgado I've removed this logic since I was detecting some issues when testing GPC. Sometimes the loading progress was not appearing at all. I've noticed the logic was affected when using low speed connection and clicking on a link before site finished loading.

Disabling worked great when testing this. And since we are just enabling GPC headers for sites that are Header consumers, the problem has low impact when it happens.